File: 백준/Bronze/30868. 개표/개표.java
import java.io.*;
import java.util.*;

public class Main {

    public static void main(String[] args) throws Exception {
        
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());

        while(N-- > 0) {
            StringBuilder sb = new StringBuilder();
            int n = Integer.parseInt(br.readLine());

            int m = n / 5;
            int r = n % 5;
            for(int i=0;i<m;i++) {
                sb.append("++++ ");
            }
            for(int i=0;i<r;i++) {
                sb.append("|");
            }
            System.out.println(sb);
        }
    }
}

File: 백준/Bronze/30868. 개표/개표.kt
fun main(args: Array<String>) = with(System.`in`.bufferedReader()) {
    val n = readLine().toInt()
  
    for(i in 1..n) {        
        val cnt = readLine().toInt()
        val sb = StringBuilder()

        val mok = cnt / 5
        val remain = cnt % 5

        for(j in 1..mok) {
            sb.append("++++ ")
        }

        for(j in 1..remain) {
            sb.append("|")
        }
        println(sb)        
    }

    
}
